README.md

```diff
@@ -16,9 +16,10 @@ This Hugo theme is a dead simple integration between [plausible.io](https://www.
 
 ```toml
 [params.plausible]
-   enable      = true
-   subdomain   = false
-   analytics   = "put-here-your-plausible-analytics-code"
+   enable      = true  # Whether to enable plausible tracking
+   domain   = "example.com"
```

Comment:
And i don't think domain is appropriate, because the "plausible name" for your site is whatever you want even if it will often be your site domain name.

Reply:
ooh, I did not know this, interesting! Probably best to wait for the outcome of that issue before renaming. In its current state, this key matches what Plausible calls it.

Reply:
Yes, I was bitten by  that too (hence my issue), and even ask them what to do to avoid breaking. Confusing.

I agree to left `domain` if we put a clear statement about what it is.